### PR TITLE
Removal of 2nd Blip for job change

### LINKED ISSUE #31: [SUGGESTION]
When changing to a different job, the old job blip stays on the map. For example, if a change from trucker to garbage: The Truck Shed blip stays on the map, but the Truck Storage is removed (which is good). I had this problem when changing from garbage to other jobs as well, but I fixed it with some change in the code.

**elseif garbageBlip ~= nil then
        RemoveBlip(garbageBlip)
    end**

I have tried to do the same with the truck job, but the code is so different.

**Summary:**
Would be nice if old job blips are removed when you change job.
This is not only for garbage and trucking but jobs in general. 😁

EDIT:
I see now that I didn't really fix the garb-job. the blip is gone when I change job, but the ped still stands there with the option to spawn garb-truck. Which shouldn't be possible when my job status says something else.

### PR DESCRIPTION
**Pull request by mrdiamonddirt**
**Describe Pull request**
Fixes Bug, where a blip doesn't get removed when changing jobs
fixes #31 

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? yes
- Does your code fit the style guidelines? yes
- Does your PR fit the contribution guidelines? yes

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -66,8 +67,10 @@ end
 local function RemoveTruckerBlips()
     if TruckVehBlip then
         RemoveBlip(TruckVehBlip)
+        RemoveBlip(TruckerBlip)
 	    ClearAllBlipRoutes()
```
Comment (tom-osborne) [inline client/main.lua:71]: Fix indentation here

File: client/main.lua [hunk 2]
```diff
@@ -66,8 +67,10 @@ end
 local function RemoveTruckerBlips()
     if TruckVehBlip then
```
Comment (tom-osborne) [inline client/main.lua:68]: Should probably check for `Truckerblip` too. Maybe separate them into separate if statements.

**Review by tom-osborne — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 1 above)
Comment (tom-osborne) [reply client/main.lua]: Still indented

**Comment by tom-osborne** [comment]
@mrdiamonddirt  https://github.com/qbcore-framework/qb-truckerjob/pull/33#discussion_r954005330 Fix this indentation and I'll merge.

File: client/main.lua (hunk 1 above)
Comment (mrdiamonddirt) [reply client/main.lua]: sorry i thought i fixed it

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**